projects/robot_arm/run.py

```diff
@@ -140,6 +140,10 @@ def make_env_all_params(rank, args):
     exp_args = exp_parser.parse_dict(CONFIGS[run_args.exp_name])
     logging_args, env_args, trainer_args, learner_args = exp_args
 
+    # Option to give a new wandb run name to the same experiment settings
+    if run_args.wandb_run_name != "":
+        run_args.exp_name = run_args.wandb_run_name
```

Comment:
Exp_name has been used to define the CONFIG previously, and we are redefining at a second point. Although it is a string and shouldn't be an issue, this pattern of redefining a variable for a different purpose should be avoided. 

Maybe a cleaner implementation would be to create a variable called exp_name or exp_log_name, and set it to `exp_log_name = run_args.wandb_run_name | run_args.exp_name`. This will default to the second or if the first is None or an empty string. Then the calls to `run_args.exp_name` can be rerouted to that new variable.

Reply:
Oh I'm sorry, I meant to write run_args.wandb_run_name = run_args.exp_name so that if there is no wandb name specified it defaults to the experiment name